vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp

```diff
@@ -414,7 +414,12 @@ $( document ).ready(function() {
   <br>
   Modified on <span class="date">${space.modificationDate}</span> by ${space.modifiedBy}.     
 </div>
-
+<div id="errorMessage" class="alert alert-danger alert-dismissible fade show" role="alert" style="display: none;">
+  <strong></strong>
+  <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+    <span aria-hidden="true">&times;</span>
+  </button>
+</div>
```

Comment:
there is an error alert box in the skeleton file. Let's use that one instead (you probably have to adjust the if structure in the skeleton, so that the alert is always there, just not always shown).

Reply:
do you still need this?